app/src/main/java/org/mozilla/reference/browser/topsites/ui/TopSitesView.kt

```diff
@@ -46,12 +44,11 @@ class TopSitesView @JvmOverloads constructor(
     }
 
     fun init(
-        loadUrlUseCase: SessionUseCases.LoadUrlUseCase,
-        tabsUseCases: TabsUseCases,
+        context: Context,
```

Comment:
Why? Didn't we passed a Context in the constructor?

Reply:
But, this context refers to the BrowserActivity

Reply:
The view context is also the activity